pages/linux/runuser.md

```diff
@@ -0,0 +1,24 @@
+# runuser
+
+> Run commands as a specific user and group IDs.
+> Do not ask for password. Only useful when run as root user.
+
+- Run 'command' under substitute 'user':
+
+`runuser {{user}} -c '{{command}}'`
+
+- Run 'command' under substitute 'user' and 'group':
+
+`runuser {{user}} -g {{group}} -c '{{command}}'`
+
+- Start a login shell as a specific user:
+
+`runuser {{user}} -l`
+
+- Specific a 'shell' for running or login instead of default shell:
+
+`runuser {{user}} -l -s /bin/sh`
+
+- Preserve the entire environment of root (if running by root). Ignore when having option `--login`:
```

Comment:
Perhaps this will read better?
```suggestion
- Inherit the current environment (ignored when using the `--login` option):
```

Reply:
The original sentence is describing exactly the concept of **preserve** env from `runuser` or even `su` philosophy, we cannot use **inherit** in this case. Disagree.